mauth_client/lambda_helper.py

```diff
@@ -20,4 +20,4 @@ def _get_private_key():
         except ModuleNotFoundError:
             pass
 
-    return private_key.replace(" ", "\n").replace("\nRSA\nPRIVATE\nKEY", " RSA PRIVATE KEY")
+    return private_key.replace("\\n", " ").replace(" ", "\n").replace("\nRSA\nPRIVATE\nKEY", " RSA PRIVATE KEY")
```

Comment:
hmm, seems like the name of the file is rather deceptive if this code it affecting things other than Lambdas? @ykitamura-mdsol